crates/astria-cli/src/bridge/collect.rs

```diff
@@ -50,6 +50,9 @@ pub(super) struct Command {
     /// The eth address of the astria bridge contracts.
     #[arg(long)]
     contract_address: ethers::types::Address,
+    /// Whether the contract is an ERC20 contract.
```

Comment:
I think the doc comment should target the user of the CLI since it's used to create the help string.  So, I'd lean more towards just "The contract is an ERC20 contract" since that's what enabling the flag represents.